packages/link/Link.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import isAbsoluteUrl from 'is-absolute-url';
+
+// if absolute, return url. otherwise loadappify the url
```

Comment:
This package is not IE11 compatible. I would submit a PR to this repo and add the file as an export and then use it here:
https://github.com/Availity/sdk-js/blob/master/packages/resolve-url/is-absolute-url.js#L2-L8

Reply:
@robmcguinness  we may just want to fork that repo for `is-absolute-url` and publish to `@availity` otherwise we can just export it from the `sdk-js` package you created for `resolve-url`

Reply:
good catch: https://github.com/Availity/sdk-js/pull/105

Reply:
6a7d02934d8cd2432fbea67eb5dc2d575411fab3